src/t2iapi/metric/service.proto

```diff
@@ -162,4 +162,13 @@ service MetricService {
    */
   rpc SetMetricCategoryMsrmtOrClc (BasicHandleRequest)
       returns (BasicResponse);
+
+  /*
+    Set the @MetricCategory to a value ∈ { Set , Preset } for the pm:AbstractMetricDescriptor with the provided handle.
+    'Set' and 'Preset' are of type pm:MetricCategory as defined in the IEEE Std 11073-10207-2017. The manipulated
+    device state shall be persistent until a next manipulation call. If the device is not able to maintain the static
+    state, it shall return RESULT_NOT_SUPPORTED.
+   */
+  rpc SetMetricCategorySetPreset (BasicHandleRequest)
```

Comment:
I would expect the same naming pattern as used for the other manipulation "SetMetricCategoryMsrmtOrClc", so pls add "Or" to the manipulation name